internal/pkg/skuba/cni/cilium.go

```diff
@@ -145,6 +187,98 @@ func CreateOrUpdateCiliumConfigMap(client clientset.Interface) error {
 	return nil
 }
 
+// MigrateEtcdToCrd performs the migration of Cilium internal data from etcd
+// cluster to CRD during upgrade from Cilium 1.5 to Cilium 1.6. This step is not
+// mandatory, without it, Cilium is going to regenerate data from scratch which
+// might result in service downtimes. If the automated migration is not
+// successful, the upgrade should still proceed with the risk of temporary
+// service downtimes.
```

Comment:
What do we do to notify the user? Are customers expected to handle this possible flakey state?
Should we have some extra healthcheck tool after the migration?

Reply:
I will add the warning to the user. There is no manual step needed if migration was not possible. If migration could not be done, then Cilium 1.6 is going to start with CRD from scratch and regenerate the data itself, but existing services can have network down for some time, aprox few minutes.

Reply:
Yeah that's what I understood and read again in this comment. I just want to make sure it's not only obvious for skuba devs, but also for deployers.